redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -320,13 +310,26 @@ internal class LazyListContainerCell(
   override fun layoutSubviews() {
     super.layoutSubviews()
 
-    val content = this.content ?: return
+    if (contentView.subviews.isEmpty()) return
+
+    val content = contentView.subviews.first() as UIView ?: return
```

Comment:
Surprisingly, yes--without it, if there is no content view to be found, this throws a `List is empty.` error that bricks the whole Treehouse app.

Reply:
You would need to use `firstOrNull()` to receive a nullable value on which the `?:` would then be able to apply. With `first()` it will throw on empty and never execute that return.

Reply:
Fixed in #2252